### DISCUSSION TITLE
Settings up social cards with netlify

### DISCUSSION THREAD

**Discussion by Mara-Li** [squidfunk/mkdocs-material#4722] [Q&A]
Hello!
To deploy my website, I use [netlify](https://www.netlify.com/).

I want to setting up [social](https://squidfunk.github.io/mkdocs-material/setup/setting-up-social-cards). In the docs, they said I must `.gitignore` my `.cache` and use it during building, with an example on github pages.

Anyone know how I can set-up the netlify build to restore the `.cache` ?

Thanks!

**Comment by MaximilianKohler** [comment]
Who is "they"? I assume you're using auto-deploy via github? Because when I manually upload mine to Netlify it works fine. I don't think you need to restore the `.cache`. I think that's only temporary. The actual images the site uses are in `\site\assets\images\social`.

**Comment by squidfunk** [comment]
Restoring `.cache` is optional – it will just speed up the build. For restoring built artifacts, it's a good idea to consult Netlify's docs, or third-party blog articles [like this one](https://mxb.dev/blog/persistent-build-folders-netlify/).
